packages/manager/core/application/src/ovh-tracking.tsx

```diff
@@ -83,16 +83,32 @@ export default function OvhTracking() {
     });
   };
 
+  const ovhTrackShadowElement = (element) => {
+    const elementInShadowRoot = element.shadowRoot.querySelector(
+      `[${'data-tracking'}]`,
+    );
+    if (
+      elementInShadowRoot &&
+      OSDS_COMPONENT.includes(elementInShadowRoot?.tagName?.toUpperCase())
+    ) {
+      const trackingValueInShadowRoot = elementInShadowRoot.getAttribute(
+        'data-tracking',
+      );
+      ovhTrackingSenClick(trackingValueInShadowRoot);
+    }
+  };
+
   const ovhTrackingAction = (event) => {
     const element = event.target as HTMLElement;
     const closestWithTracking = element.closest(`[${'data-tracking'}]`);
```

Comment:
it feels unsafe to traverse parents to find the closest "data-tracking" element, how are we sure that the element found relates to the current action ?

Reply:
the dev has to put the attribute in the right element dom or in the parent, we can improve this tracking tool with feedback 👍